src/Features/Core/Portable/Diagnostics/DiagnosticKinds.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.CodeAnalysis.Diagnostics;
 
 [Flags]
-internal enum DiagnosticKinds
+internal enum DiagnosticKind
```

Comment:
I thought flags enum should have a plural name :-) I am fine with the rename though.

Reply:
Kindly also rename the file if you are renaming the enum.

Reply:
Yes, flags should be plural